### PR TITLE
Move existing files from create folder into subfolders

### PR DESCRIPTION
**Pull request by opokornyy**
**What this PR does / why we need it**:

This PR  moves already existing files from create folder into subfolders so that each command will be in its own package.

**Release note**:
```None

```

### CODE REVIEW

File: tests/virtctl/create_vm.go [hunk 1]
```diff
@@ -237,7 +243,7 @@ func setFlag(flag, parameter string) string {
 }
 
 func runCmd(args ...string) ([]byte, error) {
-	_args := append([]string{CREATE, VM}, args...)
+	_args := append([]string{create}, args...)
```
Comment (0xFelix) [inline tests/virtctl/create_vm.go:246]: Can't you keep `VM` here? Then you don't need to add it to the runCmd calls.

Comment (opokornyy) [reply tests/virtctl/create_vm.go]: I have removed this function as we have discussed.

**Comment by 0xFelix** [comment]
Thanks!

/lgtm

**Comment by ksimon1** [comment]
/lgtm 
@xpivarc, @enp0s3 can you please check this PR?

File: tests/virtctl/create_vm.go [hunk 2]
```diff
@@ -164,8 +170,7 @@ var _ = Describe("[sig-compute][virtctl]create vm", func() {
 			preference := createPreference(virtClient)
 			pvc := createAnnotatedSourcePVC(virtClient, instancetype.Name, preference.Name)
 			userDataB64 := base64.StdEncoding.EncodeToString([]byte(cloudInitUserData))
-
-			out, err := runCmd(
+			out, err := clientcmd.NewRepeatableVirtctlCommandWithOut(create, VM,
```
Comment (xpivarc) [inline tests/virtctl/create_vm.go:173]: Why is this better?

**Review by xpivarc — COMMENTED** [review]
/approve

**Pull request merged**